lib/maschine_mk2_mikro_config.json

```diff
@@ -0,0 +1,713 @@
+{
+  "vendorId": 6092,
+  "productId": 4608,
+  "name": "Maschine mk2 mikro",
+
+  "ledBrightestValue": 63,
```

Comment:
Can you confirm that 63 is right for this instead of 127?  For my other NI hardware with RGB LED's the right value was 127 so this might leave the LEDs only ever at half brightness?  I have a (non-mikro) Maschine Mk2 in a drawer somewhere I can get out and take a look if it's likely they work the same.

Reply:
right, thanks! they were really at half brightness!